app/src/main/java/com/lagradost/cloudstream3/ui/settings/SettingsGeneral.kt

```diff
@@ -41,12 +43,13 @@ import com.lagradost.cloudstream3.utils.VideoDownloadManager
 import com.lagradost.cloudstream3.utils.VideoDownloadManager.getBasePath
 import com.lagradost.safefile.SafeFile
 
+@RequiresApi(Build.VERSION_CODES.N)
 fun getCurrentLocale(context: Context): String {
     val res = context.resources
     // Change locale settings in the app.
     // val dm = res.displayMetrics
     val conf = res.configuration
-    return conf?.locale?.toString() ?: "en"
+    return conf?.locales?.toString() ?: "en"
```

Comment:
Big no, we dont want this app only accessible for API 24. Afaik this will crash for everyone that goes into settings rn on low sdk ver